packages/adp-tooling/src/prompts/add-new-model/index.ts

```diff
@@ -0,0 +1,306 @@
+                    return oDataVersions[1].value;
+                }
+
+                return oDataVersions[0].value;
+            },
+            store: false,
+            validate: validateEmptyString,
+            guiOptions: {
+                mandatory: true,
+                hint: t('prompts.oDataServiceVersionTooltip'),
+                applyDefaultWhenDirty: true
+            }
+        } as ListQuestion<NewModelAnswers>,
+        {
+            type: 'input',
+            name: 'modelName',
+            message: t('prompts.oDataServiceModelNameLabel'),
+            guiOptions: {
+                mandatory: true,
+                hint: t('prompts.oDataServiceModelNameTooltip')
+            },
+            default: defaultSeviceName,
+            validate: (value: string) => {
+                return validatePromptModelName(value, isCustomerBase, changeFiles);
+            },
+            store: false
+        } as InputQuestion<NewModelAnswers>,
+        {
+            type: 'editor',
+            name: 'modelSettings',
+            message: t('prompts.oDataServiceModelSettingsLabel'),
+            store: false,
+            validate: validatePromptJSON,
+            guiOptions: {
+                hint: t('prompts.oDataServiceModelSettingsTooltip')
+            }
+        } as EditorQuestion<NewModelAnswers>,
+        {
+            type: 'confirm',
+            name: 'addAnnotationMode',
+            message: 'Do you want to add annotation?',
+            default: false
+        } as ConfirmQuestion<NewModelAnswers>,
+        {
+            type: 'input',
+            name: 'dataSourceName',
+            message: t('prompts.oDataAnnotationDataSourceNameLabel'),
+            validate: (value: string, answers: NewModelAnswers) => {
+                return validatePromptODataAnnotationsName(value, answers, isCustomerBase, changeFiles);
+            },
+            default: defaultSeviceName,
+            store: false,
+            guiOptions: {
+                mandatory: true,
+                hint: t('prompts.oDataAnnotationDataSourceNameTooltip')
+            },
+            when: (answers: NewModelAnswers) => answers.addAnnotationMode
+        } as InputQuestion<NewModelAnswers>,
+        {
+            type: 'input',
+            name: 'dataSourceURI',
+            message: t('prompts.oDataAnnotationDataSourceUriLabel'),
+            validate: validateEmptyString,
+            store: false,
+            guiOptions: {
+                mandatory: true,
+                hint: t('prompts.oDataAnnotationDataSourceUriTooltip')
+            },
+            when: (answers: NewModelAnswers) => answers.addAnnotationMode
+        } as InputQuestion<NewModelAnswers>,
+        {
+            type: 'editor',
+            name: 'annotationSettings',
+            message: t('prompts.oDataAnnotationSettingsLabel'),
+            validate: validatePromptJSON,
+            store: false,
+            guiOptions: {
+                mandatory: false,
```

Comment:
`mandatory: false` is the default, so it is not needed.

Reply:
Removed with: 3c30ceec2ec0bc7a970a44d3e4ef21d56c22dc2d